db/migrate/20200920161613_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :email
+      t.string :password_digest
+      t.string :api_key
```

Comment:
I looked in to storing api_key as a token rather than a string, but I couldn't get it to work properly.